packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -190,7 +190,7 @@ place to serve as replication buckets, you can supply these at Pipeline definiti
 time using the `crossRegionReplicationBuckets` parameter. Example:
 
 ```ts
-const pipeline = new codepipeline.Pipeline(this, 'MyFirstPipeline', { /* ... */ });
+const pipeline = new codepipeline.Pipeline(this, 'MyFirstPipeline', {
```

Comment:
I must fix this problem, otherwise I will never be able to pass...
something error like this:
```
    $ yarn --silent jsii-rosetta extract
    @aws-cdk.aws-codepipeline-README-snippet13.ts:3:3 - error TS0: This TypeScript feature (LabeledStatement) is not supported in examples. Please rewrite this example.

      3   crossRegionReplicationBuckets: {
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      4     'us-west-1': s3.Bucket.fromBucketAttributes(this, 'UsWest1ReplicationBucket', {
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    ...
      9     }),
        ~~~~~~~
     10   },
```